src/app/services/deposits/deposits.service.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { DepositsResponse } from 'src/app/models/Deposits';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DepositsService {
+  DepositsUrl =
+    'https://landville-backend-web-api.herokuapp.com/api/v1/transactions/my-deposit/';
```

Comment:
@Genza999, we should have all our constants in the environments file.

Reply:
Thanks @CryceTruly , I have worked on this